.github/workflows/nightly-koncur-0.9.yaml

```
@@ -198,6 +198,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Send failure data to Slack workflow
+        if: ${{ secrets.SLACK_WEBHOOK_URL != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)